docs/topic_guide/coordinates/carrington.rst

```diff
@@ -122,7 +122,7 @@ Footnotes
 
 .. [#apparent] The term "apparent" (as opposed to "true") is used to indicate that the observer perceives the locations and orientations of objects different from how they truly are due to effects that include light travel time.
 .. [#subEarth] "Sub-Earth" is a special case of "sub-observer", where the observer is at Earth (specifically, Earth center).  That is, the sub-Earth point is the point on the Sun's surface "below" the center of the Earth along the Sun-Earth line.
-.. [#Carrington] Carrington (1863), *Observations of the Spots on the Sun*, p. 16 (`<https://archive.org/details/observationsofsp00carr/page/16/mode/1up>`__) and p.27 (`<https://archive.org/details/observationsofsp00carr/page/27/mode/1up>`__)
+.. [#Carrington] Carrington (1863), *Observations of the Spots on the Sun*, `p. 16 and p.27 <https://www.google.com/books/edition/_/GY1DAAAAcAAJ>`__
```

Comment:
As @alasdairwilson [suggested](https://matrix.to/#/%23sunpy-development%3Aopenastronomy.org/%24172909985870123GnDis%3Amatrix.org?via=cadair.com&via=matrix.org&via=openastronomy.org):

```suggestion
.. [#Carrington] Carrington (1863), *Description of a Singular Appearance seen in the Sun on September 1, 1859*, `<https://adsabs.harvard.edu/full/1859MNRAS..20...13C>`__
```

?

Reply:
The title is different and it only has 4 pages, the references to other pages doesn't add up anymore. 

But either way, I don't care about citing Carrington correctly

Reply:
oh yeah this isn't the same thing.